data/companies.yml

```
@@ -284,6 +284,12 @@ integrations:
     description: "Space Cloud is a Kubernetes based serverless platform that provides instant, realtime APIs on any database, with event triggers and unified APIs for your custom business logic."
     details:
       - "Space Cloud integrates with Kubernetes and Istio natively to bring to you a highly scalable Serverless Platform. It encrypts all traffic by default and lets you describe communication policies to protect your microservices."
+  - name: "CSFMesh"
+    logo: "/logos/asiainfo.png"
+    url: "https://www.asiainfo.com/zh_cn/product_aisware_aifPaaS.html"
```

```suggestion
    url: "https://www.asiainfo.com/en_us/product_aisware_aifPaaS.html"
```
Provide an en url please.

---

ok , i already change the url